charts/xp-management/templates/service-swagger.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "management-svc.fullname" .}}-swagger
+  name: {{ template "management-svc.swagger-name" .}}
```

Comment:
Can we not use the same service name as the management service, suffixed with `-swagger` ? https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/service.yaml#L5

Reply:
While at it, can we also make the 2 service names in the `xp-treatment` chart consistent with the [management service](https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/service.yaml) ?

Reply:
> Can we not use the same service name as the management service, suffixed with -swagger ? https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/service.yaml#L5

Wait do you mean, don't use (the same service name as the management service, suffixed with -swagger)? i.e. don't use `{{ include "common.set-value" (list (include "management-svc.fullname" .) $globServiceName) }}-swagger`? Or did you mean, use it?

Reply:
> Ah I wrote this in the PR description that I'll do it in a separate PR 

My bad - I didn't read the description closely enough.

As for the Swagger name - I meant "use it" 😅 :

> {{ include "common.set-value" (list (include "management-svc.fullname" .) $globServiceName) }}-swagger

I think the one Merlin is using for Swagger is a little more complex than it needs to be - I'd prefer following the same format, using the same functions as the [service name](https://github.com/caraml-dev/helm-charts/blob/fc7832120197f2c478a10d051cc28054f6e1579d/charts/merlin/templates/merlin-service.yaml#L6), just suffixed with `-swagger`.

Reply:
Ah okay haha, alright got it, I've made the changes already :) Thanks!